tests/test_solution.py

```diff
@@ -86,5 +127,4 @@ def test_getSols():
     assert any(sol[x] == 0.0 for sol in m.getSols())
 
 if __name__ == "__main__":
```

Comment:
Let's remove this unneeded main too 😄

Reply:
I removed all of them on the other PR, that should be okay, right?

Reply:
Yup :)